surprise/prediction_algorithms/algo_base.py

```diff
@@ -26,6 +26,8 @@ class AlgoBase(object):
             baseline estimate, the ``baseline_options`` parameter is used to
             configure how they are computed. See
             :ref:`baseline_estimates_configuration` for usage.
+        verbose(bool): Whether to print trace messages of bias estimation,
```

Comment:
No need to change `AlgoBase`, I'd rather set an explicit parameter for all algorithms that make a call to `compute_similarities` or `compute baseline`.
Just like you did with `BaselineOnly`.

Basically: add the verbose option to all knns algorithms (an explicit one, no `**kwargs`) and to `BaselineOnly`, with the docstring etc...